rosdep/python.yaml

```diff
@@ -3957,16 +3957,7 @@ python-rosinstall-generator:
   rhel:
     '7': [python2-rosinstall_generator]
   ubuntu:
-    lucid: [python-rosinstall-generator]
-    maverick: [python-rosinstall-generator]
-    natty: [python-rosinstall-generator]
-    oneiric: [python-rosinstall-generator]
-    precise: [python-rosinstall-generator]
-    quantal: [python-rosinstall-generator]
-    raring: [python-rosinstall-generator]
-    saucy: [python-rosinstall-generator]
-    trusty: [python-rosinstall-generator]
-    trusty_python3: [python3-rosinstall-generator]
+    '*': [python-rosinstall-generator]
```

Comment:
Since the older distributions have all been cleaned up we can simplify this key expression to
```
ubuntu: [python-rosinstall-generator]
```